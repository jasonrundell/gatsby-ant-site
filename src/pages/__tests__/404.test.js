import React from 'react'
import renderer from 'react-test-renderer'

import Page404 from '../404'

describe('Page404', () => {
  it('renders correctly', () => {
    const data = {
      site: {
        siteMetadata: {
          title: 'Page404',
        },
      },
    }

    const tree = renderer.create(<Page404 data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
