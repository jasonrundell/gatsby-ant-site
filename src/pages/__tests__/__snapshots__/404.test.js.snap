// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        <a
          className="link"
          href="/"
        >
          Page404
        </a>
      </h1>
    </div>
  </header>,
  <div
    className="container"
  >
    <nav
      aria-label="Breadcrumb"
      className="breadcrumb"
    >
      <ol>
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/blog/"
          >
            Blog
          </a>
        </li>
      </ol>
    </nav>
  </div>,
  <div
    className="container"
  >
    <main>
      <h1>
        Not found
      </h1>
      <p>
        Sorry folks, nothing here to see.
      </p>
      <a
        href="/"
      >
        Go back to the homepage
      </a>
    </main>
    <footer
      className="footer"
    >
      Â© 
      2019
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        rel="noopener noreferrer"
      >
        Gatsby
      </a>
      .
       
      Calin says hello.
    </footer>
  </div>,
]
`;
